### PR TITLE
Add Sponsored Site page.

### PR DESCRIPTION
**Pull request by Crell**
Still pending the contact form, but reviewable for now.

### CODE REVIEW

File: src/overview/pricing/sponsored.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Sponsored Community Sites
+
+Platform.sh provides sponsored hosting for Free Software projects and tech community events and organizations as part of our effort to support the Free Software community.
+
+## Eligibility
+
+* A non-trivial software project released under a Free and Open Source license (*GPL, BSD, MIT, Apache, etc.)
```
Comment (chadwcarlson) [inline src/overview/pricing/sponsored.md]:
```suggestion
* A non-trivial software project released under a Free and Open Source license (GPL, BSD, MIT, Apache, etc.)
```

File: src/overview/pricing/sponsored.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+* A community-oriented conference or similar event run either by the community or a related non-profit corporation.  For-profit events and organizations are not eligible.
+* An offline IT community organization, such as a local user group for a Free and Open Source project.
+* An online non-profit community project that supports one of the above.
+
+The Platform.sh Developer Relations team manages the program and collectively has final say on what projects are eligible for sponsorship.
+
+Platform.sh will periodically reevaluate sponsored sites to ensure they remain eligible, and may with a month notice terminate the free service at its own will. Platform.sh may terminate this policy providing a month notice to allow users to migrate away
```
Comment (chadwcarlson) [inline src/overview/pricing/sponsored.md]:
```suggestion
Platform.sh will periodically reevaluate sponsored sites to ensure they remain eligible, and may terminate the free service at its own will. Platform.sh will provide a month notice to allow users to migrate in these cases.
```

File: src/overview/pricing/sponsored.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+The sponsored site may ask, and Platform.sh shall accept variations of the widget as long as these fall within the branding guidelines of Platform.sh.
+
+Platform.sh may announce on its own media presence both online and offline that it hosts the said project, and kindly ask the sponsored site communicate at least once over its usual social media presence (Twitter, Mailing list, etc) the sponsored hosting.
+
+If the eligible project is a registered non-profit organization in the US or Germany, it is also expected to provide paperwork to register an in-kind donation to the organization of the equivalent price of the service ($50 USD/month or the EUR equivalent), for tax purposes.
+
+Sponsored projects will receive the same level of support as any other self-service hosted customer site.
+
+## Contact
```
Comment (chadwcarlson) [inline src/overview/pricing/sponsored.md]: missing text?

File: src/overview/pricing/sponsored.md (hunk 1 above)
Comment (Crell) [reply src/overview/pricing/sponsored.md]: The * here is to indicate AGPL, LGPL, GPL, any version, etc.

File: src/overview/pricing/sponsored.md (hunk 3 above)
Comment (Crell) [reply src/overview/pricing/sponsored.md]: Waiting on the form from Chris Yates so we can embed a button for it.

File: src/overview/pricing/sponsored.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Sponsored Community Sites
+
+Platform.sh provides sponsored hosting for Free Software projects and tech community events and organizations as part of our effort to support the Free Software community.
+
+## Eligibility
+
+* A non-trivial software project released under a Free and Open Source license (*GPL, BSD, MIT, Apache, etc.)
+* A community-oriented conference or similar event run either by the community or a related non-profit corporation.  For-profit events and organizations are not eligible.
+* An offline IT community organization, such as a local user group for a Free and Open Source project.
+* An online non-profit community project that supports one of the above.
```
Comment (otaviojava) [inline src/overview/pricing/sponsored.md]: @Crell 
Can we include some social project such as https://glocal.org.br/aceleradora/

**Pull request merged**